src/RequestForm.js

```diff
@@ -425,11 +443,111 @@ class RequestForm extends React.Component {
     this.props.onSubmit(data);
   }
 
+  renderActionMenu = ({ onToggle }) => {
+    const { onCancel } = this.props;
+
+    if (!this.isEditForm()) {
+      return (
+        <Button
+          buttonStyle="dropdownItem"
+          id="clickable-cancel-new-request"
+          data-test-cancel-new-request-action
+          onClick={() => {
+            onCancel();
+            onToggle();
+          }}
+        >
+          <Icon icon="times-circle">
+            <FormattedMessage id="ui-requests.newRequest.cancel" />
+          </Icon>
+        </Button>
+      );
+    }
+
+    return (
+      <React.Fragment>
```

Comment:
you have react import already so you can just add Fragment there
like import React, { Fragment } from 'react';